Doc/library/pdb.rst

```diff
@@ -545,6 +545,14 @@ can be overridden by the local file.
 
    Print the return value for the last return of a function.
 
+.. seealso::
+
+   Module :mod:`faulthandler`
+      Used to dump Python tracebacks explicitly, on a fault, after a timeout, or on a user signal.
+
+   Module :mod:`pdb`
```

Comment:
Did you mean to add this in the traceback module or somewhere else?

Reply:
I inserted it into the traceback. It slipped my mind :))